src/include/iscsi-lio-server/UI_dialogs.rb

```diff
@@ -678,41 +639,20 @@ def contents
       VSpacing(5),
         VBox(
           VSpacing(2),
-            Left(
-              HBox(
-                HSpacing(25),
-                  @no_discovery_auth_checkbox,
-                  HSpacing(25),
-              )
-            ),
-            VSpacing(1),
-            @target_discovery_auth,
-            @initiator_discovery_auth,
-            VSpacing(2),
+          Left(
+            HBox(
+              # TODO: It should use checkbox frame
```

Comment:
I think without generating a Trello card for this at once these kind of TODOs will never be solved. We have already "enough" TODOs and FIXMEs in the code.... :-)

Reply:
we discuss that this module need serious effort to make it more readable and robust if we should maintain it. So having checkbox frame instead of checkbox that disabling field is just small TODO :)